build/shared-runtime.version

```diff
@@ -1 +1 @@
-1.2.0-beta-001304-00
\ No newline at end of file
+1.1.1;release/1.1.0,2.0.0-beta-001747-00;master
```

Comment:
The file is intentionally simple because we want it updated from another repo. Let's just install required dependencies i.e. `1.1.1` explicitly and use this file to specify the shared runtime that the gate branch tried out.